pallets/process-validation/src/tests/create_process.rs

```diff
@@ -150,79 +152,79 @@ fn updates_version_correctly_for_new_process_and_dispatches_event() {
     });
 }
 
-#[test]
-fn combined_with_depth_succeeds() {
-    new_test_ext().execute_with(|| {
-        System::set_block_number(1);
-        assert_ok!(ProcessValidation::create_process(
-            Origin::root(),
-            PROCESS_ID1,
-            bounded_vec![Combined {
-                operator: BinaryOperator::AND,
-                restriction_a: { Box::new(None) },
-                restriction_b: Box::new(Combined {
-                    operator: BinaryOperator::AND,
-                    restriction_a: { Box::new(None) },
-                    restriction_b: { Box::new(None) }
-                })
-            }]
-        ));
-        let expected = TestEvent::ProcessValidation(ProcessCreated(
-            PROCESS_ID1,
-            1u32,
-            bounded_vec![Combined {
-                operator: BinaryOperator::AND,
-                restriction_a: { Box::new(None) },
-                restriction_b: Box::new(Combined {
-                    operator: BinaryOperator::AND,
-                    restriction_a: { Box::new(None) },
-                    restriction_b: { Box::new(None) }
-                })
-            }],
-            true
-        ));
-        // sets version to 1 and returns true to identify that this is a new event
-        assert_eq!(<VersionModel<Test>>::get(PROCESS_ID1), 1u32);
-        assert_eq!(System::events()[0].event, expected);
-    });
-}
+// #[test]
```

Comment:
Any reason this is now a giant comment?

Reply:
Disables the tests correspoonding to the disabled combined restriction